### DISCUSSION TITLE
[Live Competition] My agent disappeared from the Live Competition despite I don't nothing.

### DISCUSSION THREAD

**Discussion by Masaty99** [yasserfarouk/scml#120] [Q&A] [answered]
Dear @yasserfarouk. 

Good morning. My name is Masato Kijima from Tokyo University of Agriculture and Technology. 
I participate in SCML OneShot track. 
I found several agents, including my agent named KanbeAgent, have suddenly disappeared from the Live Competition after last tournament finished. 
I have got an email that the Agent Test Success about 10 minutes after the last competition held despite I didn't update my agent. Is this relevant to this problem?
To tell the truth, a similar phenomenon occurred the day before yesterday. My agent has also disappeared from the Live Competition after I have got three emails that the Agent Test Success at the same time despite I didn't update my agent. 

Please tell me what is happening in the Live Competition. If there is a problem with my agent, please let me know. 

Best Regards,
Masato Kijima

**Comment by yasserfarouk** [comment, marked as answer]
Dear @Masaty99 

The agent was marked as `disabled`. This should not have happened except if you explicitly did that in the agent's page. We will investigate the reason behind this mishap and correct for it in the future.
For now, I reenabled all missing agents and your agent is one of them.

**Reply by Masaty99** [reply]
Dear @yasserfarouk 

Thank you for your investigation. I understand why my agent was missed. 
Thank you for your prompt and courteous help.

**Reply by yasserfarouk** [reply]
Just in case you face the same issue again, please login to your account, go to `Your Home` and in the `Active Agent` page (the default), press `Allow the agent to enter the official ANAC competition`.

**Reply by Masaty99** [reply]
Thank you for your kind help. 
If I face the same issue again, I do so.

**Reply by Masaty99** [reply]
I report that I face the same issue again.
Several agents has disappeared again as it has occurred yesterday.
My agent was able to get it back by using the method you described to me yesterday.
I also confirmed my latest agent code is uploaded in the competition by pressing `Download Code` after pressing `Allow the agent to enter the official ANAC competition`. 
You may know this, but I report it just in case.
Sorry for the frequent reports, but thank you for your cooperation.

**Reply by yasserfarouk** [reply]
Thank you very much for reporting. Yes, we are facing an issue with our server. We returned the status for all submitted agents to their normal state and will confirm participation using another form today.
We apologize for the convenience and confirm that your agent is running in  the competition.

**Reply by Masaty99** [reply]
Thank you for your prompt and courteous help. 
I confirmed  that my agent is running in the competition.
